Dockerfile

```diff
@@ -1,4 +1,6 @@
-FROM nginx:1.17.7-alpine
+FROM nginx:1.19.5-alpine
+
+ENV NGINX_ENTRYPOINT_QUIET_LOGS=1
```

Comment:
See [docs](https://github.com/docker-library/docs/tree/a8690c3dd861de70c3d572f8e838818b02fc42b9/nginx#entrypoint-quiet-logs) regarding this.

If we don't have this we'll get the following logs on startup:
```
nginx_1  | /docker-entrypoint.sh: /docker-entrypoint.d/ is not empty, will attempt to perform configuration
nginx_1  | /docker-entrypoint.sh: Looking for shell scripts in /docker-entrypoint.d/
nginx_1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/10-listen-on-ipv6-by-default.sh
nginx_1  | 10-listen-on-ipv6-by-default.sh: Getting the checksum of /etc/nginx/conf.d/default.conf
nginx_1  | 10-listen-on-ipv6-by-default.sh: info: /etc/nginx/conf.d/default.conf differs from the packaged version
nginx_1  | /docker-entrypoint.sh: Launching /docker-entrypoint.d/20-envsubst-on-templates.sh
nginx_1  | /docker-entrypoint.sh: Configuration complete; ready for start up
```